[All Hosts] (debugging) updating the article for debugging with developer tools (#3005)
OfficeDev/office-js-docs-pr: docs/includes/identify-webview-in-ui.md

> In recent versions of Office, you can identify the webview control that is being used through the [personality menu](../design/task-pane-add-ins.md#personality-menu) on any add-in. Open the menu and select **Security Info**. On Windows, the **Security Info** dialog, the **Runtime** will report **Microsoft Edge**, **Microsoft Edge Legacy**, or **Internet Explorer**. The runtime is not included on the dialog in older versions of Office. 

Reviewer: I don't think the personality menu is available for Outlook add-ins so saying "on any add-in" is not true. Maybe we can tweak it to something like "on any add-in where it's available"?

We may also need to update the personality menu section to say that it's not available for Outlook.
```suggestion
In recent versions of Office, you can identify the webview control that is being used through the [personality menu](../design/task-pane-add-ins.md#personality-menu) on any add-in. Open the menu and select **Security Info**. In the **Security Info** dialog on Windows, the **Runtime** reports **Microsoft Edge**, **Microsoft Edge Legacy**, or **Internet Explorer**. The runtime isn't included on the dialog in older versions of Office. 
```
